src/base-command.ts

```diff
@@ -115,15 +115,15 @@ export default abstract class BaseCommand extends Command {
 
   async catch(error: any): Promise<void> {
     if (error.oclif && error.oclif.exit === 0) return;
-    process.exitCode = 1;
 
     try {
       if (error.stack) {
         error.stack = [...new Set(error.stack.split('\n'))].join("\n");
       }
 
       if (error instanceof ValidationErrors) {
-        return prettyValidationErrors(error);
+        prettyValidationErrors(error);
```

Comment:
Can we add a test to the validation oclif command test? That way we ensure that pretty printing the validation errors will always exist 1. 

Also it would be good to ensure that the output is actually pretty and doesn't include a stack trace.

Reply:
The output to console from super.catch is suppressed and does not print the stacktrace.  I'll write more test cases.

Reply:
added test cases to the validation oclif command test to test stacktrace is suppressed, exitCode is 1, and validation is pretty printed;